Library/Homebrew/software_spec.rb

```diff
@@ -449,10 +454,10 @@ def root_url(var = nil, specs = {})
   def path_resolved_basename(name, checksum, filename)
     if root_url.match?(GitHubPackages::URL_REGEX)
       image_name = GitHubPackages.image_formula_name(name)
-      ["#{image_name}/blobs/sha256:#{checksum}", filename]
+      ["#{image_name}/blobs/sha256:#{checksum}", filename&.github_packages]
     else
       # TODO: this can be removed when we no longer use Bintray
-      filename
+      filename&.bintray
```

Comment:
I am using an internal tap and I think because of the change in this PR, I am now getting the following output in `brew info --json=v1 <formula>`:
```
...
    "bottle": {
      "stable": {
        "rebuild": 0,
        "root_url": "https://<domain>/homebrew/bottles",
        "files": {
          "catalina": {
            "cellar": "/usr/local/Cellar",
            "url": "https://<domain>/homebrew/bottles/",
            "sha256": "..."
          }
        }
      }
    },
 ...
```

Notice the bottle's URL is missing the actual full path and only contains the base URL. Is there a way we can restore this for third-party taps?